pos_order_mgmt/models/pos_order.py

```diff
@@ -88,7 +88,7 @@ def _prepare_done_order_line_for_pos(self, order_line):
     def _prepare_done_order_payment_for_pos(self, payment_line):
         self.ensure_one()
         return {
-            'statement_id': payment_line.statement_id.id,
+            'journal_id': payment_line.journal_id.id,
```

Comment:
Why changing this?

Reply:
Well, makes sense since it's the right reference to match in the reprinted ticket.

Reply:
OK